src/qibo/optimizers.py

```diff
@@ -56,8 +56,16 @@ def newtonian(loss, initial_parameters, method='Powell', options=None):
         options (dict): Dictionary with options accepted by
             ``scipy.optimize.minimize``.
     """
-    from scipy.optimize import minimize
-    m = minimize(loss, initial_parameters, method=method, options=options)
+    if method == 'parallel_lbfgsb':
+        from qibo.config import raise_error, get_backend, THREADS
+        if get_backend() == 'custom' and THREADS.get("intra") != 1:
```

Comment:
In principle we could set the intra threads to 1 automatically here instead of raising an error, right? On the other hand, it may not be a good idea to modify things under-the-hood without the user knowing.